systemtest/src/main/java/io/strimzi/systemtest/resources/operator/SetupClusterOperator.java

```diff
@@ -190,15 +191,16 @@ public SetupClusterOperatorBuilder defaultInstallation() {
      * Don't use this method in tests, where specific configuration of CO is needed.
      */
     public SetupClusterOperator runInstallation() {
-        LOGGER.info("Cluster operator installation configuration:\n{}", this::toString);
+        LOGGER.info("Cluster operator installation configuration:\n{}", this::prettyPrint);
+        LOGGER.debug("Cluster operator installation configuration:\n{}", this::toString);
```

Comment:
Is this needed here? We have almost the same for the info log level

Reply:
Discussed offline